ui/imports/shared/controls/chat/ChatInputLinksPreviewArea.qml

```diff
@@ -4,26 +4,35 @@ import QtGraphicalEffects 1.15
 import QtQuick.Layouts 1.15
 
 import StatusQ.Core 0.1
+import StatusQ.Popups 0.1
 
 import shared.status 1.0
 import shared.controls.chat 1.0
 
+import utils 1.0
+
 import SortFilterProxyModel 0.2
 
 Control {
     id: root
 
     required property var imagePreviewModel
     required property var linkPreviewModel
+    required property bool showLinkPreviewSettings
 
     readonly property alias hoveredUrl: d.hoveredUrl
-    readonly property int contentItemsCount: imagePreviewModel.length + d.filteredModel.count
+    readonly property bool hasContent: imagePreviewModel.length || showLinkPreviewSettings || linkPreviewRepeater.count
```

Comment:
More explicit?

```suggestion
    readonly property bool hasContent: imagePreviewModel.length > 0 || showLinkPreviewSettings || linkPreviewRepeater.count
```

Reply:
It's JavaScript, `linkPreviewRepeater.count` is fine? ;)